### DISCUSSION TITLE
Open WebUI cannot work behind a Privoxy server

### DISCUSSION THREAD

**Discussion by zx2015** [open-webui/open-webui#6747] [General]
# Bug Report

## Installation Method

I utilized Docker to download the latest Open-WebUI image and execute it.

## Environment

- **Open WebUI Version: v0.3.35
- **Operating System: 
- ***Client - Windows 11 & Mac OS 14
- *** Server - Ubuntu 24
- **Browser (if applicable): Microsoft Edge

**Confirmation:**

- [X] I have read and followed all the instructions provided in the README.md.
- [X] I am on the latest version of both Open WebUI and Ollama.
- [] I have included the browser console logs.
- [] I have included the Docker container logs.
- [X] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

The browser can open the Open-WebUI page normally with privoxy proxy server.

## Actual Behavior:

With proxy server, Edge browser can open the Open-WebUI login page, but after login, the page becomes blank. Refreshing the page does not resolve the issue.

## Description

**Bug Summary:**
I am working in a LAN network that requires a proxy server configuration for network access. The Open-WebUI server, located in a different zone, needs to be accessed via the proxy. The proxy server utilizes Privoxy.

When my laptop is connected to the same zone as the Open-WebUI server and I remove the proxy setting from Edge, allowing it to connect directly to the server, the Open-WebUI functions as expected. I am able to log in, submit questions, and receive feedback without any issues.

## Reproduction Details

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots/Screen Recordings (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by zx2015** [comment]
I resolved the issue by removing all filters for open-webui server in the Privoxy configuration.

**Reply by ExHo7** [reply]
Hello, what configuration file(s) I've you modify in Privoxy please ?

**Reply by ExHo7** [reply]
ok "user.action"
